docs/guides/deployment/kubernetes.md

```diff
@@ -453,7 +459,7 @@ The directory should now look something like this:
 ### Tag images with a Git SHA
 
 If your code is stored in a GitHub repository, it's good practice to tag your images with the Git SHA of the code used to build it.
-This can be done in the `prefect.yaml` file with a few minor modifications.
+This can be done in the `prefect.yaml` file with a few minor modifications, and isn't yet an option with the Python deployment creation method.
```

Comment:
I think we should be saying something like "Python-based flow deployment creation method".